docs/content/FAQ/_index.md

```diff
@@ -38,17 +39,26 @@ The Infection Monkey agent shuts off either when it can't find new victims or it
 
 ## How do I reset the Monkey Island password?
 
-When you first access the Monkey Island server, you'll be prompted to create an account. If you forget the credentials you entered, or just want to change them, you need to alter the `server_config.json` file manually.
+When you first access the Monkey Island server, you'll be prompted to create an account.
+If you forget the credentials you entered, or just want to change them, you need to alter
+the `server_config.json` file manually.
 
-On Linux, this file is located at `/var/monkey/monkey_island/cc/server_config.json`. On Windows, it's based on your install directory (typically it is `C:\Program Files\Guardicore\Monkey Island\monkey_island\cc\server_config.json`). Reset the contents of this file leaving the **deployment option unchanged** (it might be "VMware" or "Linux" in your case):
+On Linux, the default path to `server_config.json` is `$HOME/.monkey_island/server_config.json`.
+On Windows, it is `%AppData%\monkey_island\server_config.json`.
+Reset the contents of this file leaving the **deployment option unchanged**
+(it might be "VMware" or "Linux" in your case):
 
 ```json
 {
-  "server_config": "password",
-  "deployment": "windows"
+  "environment": {
+    "server_config": "password",
+    "deployment": "windows"
+  }
 }
 ```
- Then, reset the Monkey Island process. Use `sudo systemctl restart monkey-island.service` on Linux or, on Windows, restart program.
+ Then, reset the Monkey Island process.
+ On Linux, use `sudo systemctl restart monkey-island.service`.
+ On Windows, restart the program.
```

Comment:
Why so many new lines (here and in other places)? They don't make a difference.

Reply:
Exactly, they don't make a difference. So as with code, we should not exceed maximum line length for readability